packages/model-viewer/src/three-components/gltf-instance/gltf-2.0.ts

```diff
@@ -1,3 +1,5 @@
+import {TextureFilter, Wrapping} from 'three';
```

Comment:
Can we remove this? I don't like the idea of glTF being dependent on three.js. I believe we're already making the assumption that these filter constants are consistent between glTF and three, which I don't love, but is probably fine since they both come from WebGL.

Reply:
That was my thinking as well, but given my change it needed that to be able to understand the type conversions between TextureFilter and MagFilter for instance.